frontend/src/js/model/node.ts

```diff
@@ -96,3 +107,23 @@ export function nodeIsAllowlisted(
 export function nodeIsElement(node: ConceptT): node is ConceptElementT {
   return "tables" in node;
 }
+
+export function getNodeIcon(
```

Comment:
helper to make sure we're using the same logic for displaying the right icon for a node